elastic/kibana
# [Deployment Management] Add landing page redirect feature and implement in security solution

Fixes https://github.com/elastic/kibana/issues/161000

## Summary

The `serverless-security` solution will have their own management landing page that lives in a different URL as the management pages for oblt and search. To streamline access to all management pages through a single URL, we have implemented an API that allows the security solution to specify a different URL for redirection when required. This ensures a seamless user experience while navigating between various management interfaces.

As a placeholder for now I've set the redirect for the security landing page to:

```typescript
management.setLandingPageRedirect('/app/security/manage');
```

##### How to test
* Boot up the security solution and make sure that going to `http://localhost:5601/ekv/app/management` will redirect you to `http://localhost:5601/ekv/app/security/manage`.
* Boot up any other solution and make sure that `http://localhost:5601/ekv/app/management` takes you to the managements default landing page.
